integration/heapster_api_test.go

```diff
@@ -53,16 +54,32 @@ var (
 
 func deleteAll(fm kubeFramework, ns string, service *kube_api.Service, rc *kube_api.ReplicationController) error {
 	glog.V(2).Infof("Deleting rc %s/%s...", ns, rc.Name)
-	if err := fm.DeleteRC(ns, rc); err != nil {
-		glog.V(2).Infof("Failed to delete rc: %v", err)
-		return err
+	retries := 5
+	for {
```

Comment:
This is a temporary fix for unreliable delete - see #559. Delete should mean delete and there should be no reason for retries. I will refactor this completely in the next PR.